### PR TITLE
dockerize linting task

### PR DESCRIPTION
**Pull request by PankhudiB**
Dockerize the linter #223

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -40,7 +40,8 @@ ci:
 # lint runs the linter.
 .PHONY: lint
 lint:
-	golangci-lint run --config .golangci.yml
+	docker rmi -f gidari_lint || true
```
Comment (prestonvasquez) [inline Makefile:43]:
```suggestion
	docker-compose -f "docker-compose.yml" up -d \
                --remove-orphans \
                --force-recreate \
                --build lint
```

**Review by prestonvasquez — CHANGES_REQUESTED** [review]
Outstanding, it works like a charm! Just one suggestion :)

**Comment by prestonvasquez** [comment]
@PankhudiB Would you mind also updating the docs/CONTRIBUTIONS.md file to remove the golangci-lint dependency requirement?

File: Makefile (hunk 1 above)
Comment (PankhudiB) [reply Makefile:43]: Thats a nice suggestion. Thank you !

**Comment by prestonvasquez** [comment]
@PankhudiB Can you resolve the merge conflicts here? Then I can merge the PR.

**Comment by PankhudiB** [comment]
@prestonvasquez Merged them, can you kindly review it ?

**Comment by prestonvasquez** [comment]
Sorry for the late reply. Thank you so much for this contribution!

**Pull request merged**